src/grisp_i2c.erl

```diff
@@ -0,0 +1,174 @@
+% @doc GRiSP I²C API.
+%
+% <a href="https://en.wikipedia.org/wiki/I²C">Inter-Integrated Circuit
+% (I²C)</a> is a synchronous addressable serial communication bus. Two types of
+% nodes can exist, controllers and targets. Multiple controllers and targets
+% can exist on the bus simultaneously. A GRiSP board acts as a controller to
+% any connected target peripherals. The controller generate the clock signal
+% and initiate communication with targets. Targets follow the clock and
+% respond when addressed.
```

Comment:
There is a missing -s at the end of controller, it's plural now, isn't it? Hence, I would write:
> Controllers generate...

Reply:
Ah, misinterpreted your suggestion.